helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -1209,7 +1209,10 @@ operator:
         tag: v1.5.0
         env:
           - name: OTEL_DOTNET_AUTO_PLUGINS
-            value: Splunk.OpenTelemetry.AutoInstrumentation.Plugin, Splunk.OpenTelemetry.AutoInstrumentation
+            value: Splunk.OpenTelemetry.AutoInstrumentation.Plugin,Splunk.OpenTelemetry.AutoInstrumentation
+          # TODO: Decide to remove or keep this env var, might be a temporary work around
```

Comment:
Same as above: I think after the `name` the comment will be more readable.

Reply:
Done. Will see if we can address the issue of having to include `OTEL_DOTNET_AUTO_RULE_ENGINE_ENABLED` before merging in this change. I'd like to avoid adding this env var is possible.

Reply:
Have a fix or work around in place for this TODO. May need to follow up more with OTL-2843. Resolving this thread for now.